lib/feature/home_page/widgets/pokemon_tile.dart

```diff
@@ -36,14 +35,22 @@ class _PokemonTileState extends State<PokemonTile> {
                   url: e['type']['url'],
                 ))).toList(),
           );
-        });
-      } else {
-        print('error');
-      }
+          if (mounted) {
+            setState(() {});
+          }
+        } else {
+          print('error');
+        }
+      });
     });
     super.initState();
   }
 
+  @override
+  void dispose() {
```

Comment:
no need for the dispose override. because you're not doing anything inside of it